docs/guides/sss_override.rst

```diff
@@ -0,0 +1,30 @@
+Local override (sss_override) users and groups
```

Comment:
Correct me if I am wrong, but isn't "Local overrides **of** users and groups" better?

Reply:
It is, but I thought having the CLI command in the title is helpful, the class name is probably sufficient.